Workbooks/Virtual Machines - Network Dependencies/Connections Overview/Connections Overview.workbook

```diff
@@ -733,37 +815,101 @@
       "content": {
         "version": "KqlParameterItem/1.0",
         "query": "",
-        "crossComponentResources": [],
+        "crossComponentResources": [
+          "{Workspace}"
+        ],
         "parameters": [
           {
-            "id": "4c14e89e-dbd3-421b-a041-c8fc87ce6e43",
+            "id": "7e4c3d29-2be6-4288-903b-95502ddab577",
             "version": "KqlParameterItem/1.0",
-            "name": "SourceComputer",
+            "name": "ComputerName",
             "type": 1,
-            "isRequired": true,
-            "query": "let row = dynamic({ConnectionGrid});\r\nlet computer = tostring(row.Key);\r\nrange i from 1 to 1 step 1\r\n| project computer",
+            "isRequired": false,
+            "query": "// {Workspace:label}\r\nlet row = dynamic({ConnectionGrid});\r\nlet computerName = row.ComputerName;\r\nprint computerName",
```

Comment:
I see this comment "// {Workspace:label}" in almost all queries. Do you need that?

Reply:
Removed, they were originally added as a _safe_ workaround to guarantee the template would initialize properly under certain conditions but looks like that has been fixed